listenbrainz/mbid_mapping/mapping/mb_metadata_cache.py

```diff
@@ -488,3 +663,27 @@ def create_mb_metadata_cache(use_lb_conn: bool):
 
         cache = MusicBrainzMetadataCache(mb_conn, lb_conn)
         cache.run()
+
+
+def incremental_update_mb_metadata_cache(use_lb_conn: bool):
+    """ Update the MB metadata cache incrementally """
+    psycopg2.extras.register_uuid()
+
+    with psycopg2.connect(config.MBID_MAPPING_DATABASE_URI) as mb_conn:
+        lb_conn = None
+        if use_lb_conn and config.SQLALCHEMY_TIMESCALE_URI:
+            lb_conn = psycopg2.connect(config.SQLALCHEMY_TIMESCALE_URI)
+
+        cache = MusicBrainzMetadataCache(mb_conn, lb_conn)
+        if not cache.table_exists():
+            log("mb metadata cache: table does not exist, first create the table normally")
+            return
+
+        log("mb metadata cache: starting incremental update")
+
+        # TODO: Update logic to get last update timestamp
```

Comment:
TODO: Decide the schema for a PG table to store the last update timestamp.

Reply:
Why should it be different from the one we use everywhere else?

Reply:
Mostly because we want to create a single table and reuse it for all cases going forward instead of creating a new table for each use case.